packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -95,3 +129,37 @@ export function parseEnvironment(values: ENVIRONMENT_MAP): ENVIRONMENT {
 
   return environment;
 }
+
+let lastMock: ENVIRONMENT_MAP = {};
+
+/**
+ * Mocks environment used for tests.
+ */
+export function mockEnvironment(values: ENVIRONMENT_MAP) {
```

Comment:
@dyladan ^^

Reply:
So the idea was to use it in multiple test files. While moving it here and exporting it so that multiple test files could use it seems to work it might not be the best solution.

We had a test-utils package but that was moved to contrib repo I believe. Should I copy paste these functions in both test files?

Reply:
but it was inside file `/test/utils/environment.test.ts ` which is perfectly fine, by moving this to `environment.ts ` mean we are exporting a test function as our api which is not something we want

Reply:
test-utils is in the contrib repo because it was only used by contrib plugins, but it might be a good idea to move it back into the main repo. I have said on other PRs I don't like the idea of exporting testing/fake/mock behavior from modules because users _will_ find them and they _will_ use them even if we dont want them to, and they will be confused and we will have to deal with it.

Reply:
so you can still easily share this function between tests in the same package, just don't export them as our sdk

Reply:
Thanks I will change accordingly as not to export the function

Reply:
Alright, exported from its original location (test file itself) not sure why i did not do that first time around, think I was fatigued at that time.

Reply:
just fix the conflicts pls